### DISCUSSION TITLE
How can I use range with DateTimePicker just like we have DatePickerInput

### DISCUSSION THREAD

**Discussion by x-08** [mantinedev/mantine#4284] [Q&A]
Hi Team,

I want to use range with DateTimePicker just Like with DatePickerInput but i cant find "type" props for DateTimePicker components.
Any Ideas or workaround that i can use?

**Comment by rtivital** [comment]
DateTimePicker does not support dates range

**Reply by zdenkolini** [reply]
is there a reason for why not?
currently I am using 2 `DateTimePicker` components to filter data in my table and think that a range option would be better UX.
found an example here, the design is a bit lackluster, but the functionality seems fine: https://www.daterangepicker.com/#example2

**Reply by rtivital** [reply]
The component does not support this feature because it has not been implemented.

**Reply by zdenkolini** [reply]
yes, I understood that, just interested if the feature is on the roadmap?
if there is interest in this being a feature, I'd be happy to give it a go after discussing the details of implementation.

**Reply by rtivital** [reply]
It is in the backlog, but I'd rather work on it myself

**Comment by gtauzin** [comment]
Would be great of this was available :)
